browser_bot.py

```
@@ -0,0 +1,604 @@
+    def __init__(self, applicant: ApplicantInfo, config: BookingConfig):
+        self.applicant = applicant
+        self.config = config
+        self.browser: Optional[Browser] = None
+        self.context: Optional[BrowserContext] = None
+        self.page: Optional[Page] = None
+        self.notified_slots: set = set()
+
+    async def setup_browser(self):
+        """Initialize the browser with anti-detection settings"""
+        if not PLAYWRIGHT_AVAILABLE:
+            raise RuntimeError("Playwright is not installed")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)